tests/torchtune/modules/optim/test_cpu_offload.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+
+import pytest
+import torch
+from torch import nn
+from torchtune.modules.optim import CPUOffloadOptimizer
+from torchtune.modules.optim._cpu_offload import _CPU_OFFLOAD_OPTIM_AVAILABLE
+
+if not _CPU_OFFLOAD_OPTIM_AVAILABLE:
+    pytest.skip("CPU offload optimizer is not available", allow_module_level=True)
+
+
+class TestCPUOffloadOptimizer:
```

Comment:
should we add a test for memory saving? Check: https://github.com/pytorch/pytorch/blob/52ad41a9c5c9cb44c54cc632433c6a5bbb23b37a/test/inductor/test_scatter_optimization.py#L184